### PR TITLE
feat: [CDS-34707]: add Serverless Aws Lambda Deploy and Serverless Aws Lambda Rollback steps

### PR DESCRIPTION
**Pull request by chetanvekariya-harness**
##### Summary:
- Add Serverless Aws Lambda Deploy and Serverless Aws Lambda Rollback steps when serverless aws lambda deployment type is selected in service tab

##### Jira Links:
https://harness.atlassian.net/browse/CDS-34707

##### Screenshots:
Screenshot 2022-03-22 at 7 05 28 PM

Screenshot 2022-03-22 at 7 05 48 PM

Screenshot 2022-03-22 at 7 06 19 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by chetanvekariya-harness** [comment]
dockerize

File: src/modules/70-pipeline/components/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/Artifactory/Artifactory.tsx [hunk 1]
```diff
@@ -70,6 +70,19 @@ function NoRepositoryResults({ error }: { error: GetDataError<Failure | Error> |
...
 
+const getRepositoryValue = (
+  formData: ImagePathTypes & { connectorId?: string },
+  isServerlessDeploymentTypeSelected = false
+): string => {
+  if (isServerlessDeploymentTypeSelected) {
+    if (formData?.repository?.value) {
+      return formData?.repository?.value
+    }
+    return formData?.repository
```
Comment (YounixIO) [inline src/modules/70-pipeline/components/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/Artifactory/Artifactory.tsx:81]: This line of code can be removed.

**Review by YounixIO — DISMISSED** [review]
LGTM

File: src/modules/70-pipeline/components/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/Artifactory/Artifactory.tsx (hunk 1 above)
Comment (chetanvekariya-harness) [reply src/modules/70-pipeline/components/ArtifactsSelection/ArtifactRepository/ArtifactLastSteps/Artifactory/Artifactory.tsx:81]: Right. I will remove it in next PR or this one if I have to push more changes.

**Review by PratyushGar — APPROVED** [review]

**Comment by chetanvekariya-harness** [comment]
dockerize

**Review by YounixIO — APPROVED** [review]

**Review by PratyushGar — APPROVED** [review]

**Comment by chetanvekariya-harness** [comment]
trigger cypress

**Pull request merged**